### DISCUSSION TITLE
eartrumpet doesn't open on boot? i'd like that added to it's settings

### DISCUSSION THREAD

**Discussion by BardsDirge** [File-New-Project/EarTrumpet#1053] [Q&A] [answered]
eartrumpet doesn't open on boot? i'd like that added to it's settings.

...or is it already able to stay active and i'm just doing something wrong?
i open the app and i have 2 volume icons in my bar.

 it works great. i love it already.
 i just want to _**not**_ need to open it.

thank you in advance for your time.

**Comment by riverar** [comment, marked as answer]
@BardsDirge EarTrumpet automatically starts with Windows 10 and Windows 11. You can verify this by looking in Task Manager > Startup. It should look like this:

![image](https://user-images.githubusercontent.com/475132/152710215-1d0880fa-4ad9-4d51-a68a-e3b2ea6c5159.png)

**Reply by BardsDirge** [reply]
i agree, it should... mines broken somehow. :(

**Comment by BardsDirge** [comment]
solved.
scoop install ignored that part apparently.

**Reply by riverar** [reply]
Thanks for the heads up. Scoop is not an official installation channel for EarTrumpet so you're on your own. But I will file an issue over there to hopefully get things fixed up!
